packages/sdks/src/functions/get-content/generate-content-url.test.ts

```diff
@@ -32,4 +33,56 @@ describe('Generate Content URL', () => {
     });
     expect(output).toMatchSnapshot();
   });
+
+  test('generate content url with apiVersion as v2', () => {
+    const output = generateContentUrl({
+      apiKey: testKey,
+      model: testModel,
+      query: { id: testId },
+      options,
+      apiVersion: 'v2',
+    });
+    expect(output).toMatchSnapshot();
+  });
+
+  test('generate content url with apiVersion as v3', () => {
+    const output = generateContentUrl({
+      apiKey: testKey,
+      model: testModel,
+      query: { id: testId },
+      options,
+      apiVersion: 'v3',
+    });
+    expect(output).toMatchSnapshot();
+  });
+
+  test('throw error when trying to generate content url with apiVersion as v1', () => {
+    expect(() => {
+      generateContentUrl({
+        apiKey: testKey,
+        model: testModel,
+        query: { id: testId },
+        options,
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore Need this ignore as TS itself was complaining about invalid type
+        apiVersion: 'v1',
```

Comment:
`@ts-ignore` is excessive because it will blindly ignore any TS error in the line below. A better approach is to type-cast the variable to what you want it to be:
```suggestion
        apiVersion: 'v1' as GetContentOptions['apiVersion'],
```